### DISCUSSION TITLE
SK memory connector to postgres

### DISCUSSION THREAD

**Discussion by jamesmkfoo23** [microsoft/semantic-kernel#6988] [Q&A] [answered]
Hi, is postgres supported with Python?

Azure docs for Azure PostgreSQL Flexible python shows the usage psycopg2
https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/connect-python?tabs=cmd%2Cpasswordless#create-a-table-and-insert-data . I believe therefore this should be the correct library psycopg2.

Trying to use the connector but i'm getting this error when i try to import the postgres memory store
```
from semantic_kernel.connectors.memory.postgres import PostgresMemoryStore

semantic_kernel\connectors\memory\postgres\__init__.py:3
  from semantic_kernel.connectors.memory.postgres.postgres_memory_store import (PostgresMemoryStore)

semantic_kernel\connectors\memory\postgres\postgres_memory_store.py:9
  from psycopg import Cursor
  
ModuleNotFoundError: No module named 'psycopg'
```

Is there any samples for SK pg memory connector?

**Comment by moonbox3** [comment]
@eavanvalkenburg thoughts here?

**Comment by eavanvalkenburg** [comment, marked as answer]
that is very interesting, @jamesmkfoo23 I haven't touched that code in a long time, I'm a bit hesitant to go in now, since we are focused on the new version of the memory connectors (see #7614), but I will add a note to the issue for that to update it to this one, if you really need it right now, then probably it's easiest to take the existing code, copy it and make any needed changes locally and off course feel free to make a PR from that as well!
